docs/guides/operator/installation.adoc

```diff
@@ -21,6 +21,66 @@ The recommended way to install the {project_name} Operator in Kubernetes environ
 
 * Be sure that you have cluster-admin permission or an equivalent level of permissions granted by an administrator.
 
+[WARNING]
+====
+*Important: Automatic OLM Upgrades*
+
+By default, OLM automatically updates the {project_name} Operator when a new version is released. This can cause several significant issues:
+
+* When using the default {project_name} image, the Operator by default uses a matching image of the corresponding {project_name} version, resulting in *unintended {project_name} upgrades* when the Operator is upgraded.
+
+* Even when using custom images, major Operator upgrades can introduce compatibility issues with your existing Keycloak CR configuration.
+
+*Potential risks:*
+
+* Unexpected downtime during automatic upgrades
+* Database schema migrations that cannot be easily rolled back
```

Comment:
In particular based upon recent issues, you cannot even reliably simply roll back micro version changes - that is once 26.x.n is applied, if you want to go back to 26.x.(n -1) then you need to restore from a database snapshot, rather than just going back 1 micro in operator / keycloak versions.

Reply:
We could possibly reword it as:
```suggestion
* No option to downgrade to the previous {project_name} version due to database schema migrations
```
@Pepo48 @shawkins WDYT?